ivy_tests/test_ivy/test_frontends/test_torch/test_other_operations.py

```diff
@@ -0,0 +1,31 @@
+import ivy
+from hypothesis import strategies as st
+from ivy_tests.test_ivy.helpers import 
+from ivy_tests.test_ivy.helpers import (
+    handle_frontend_test,
+    ivy_array_equal,
+    dtypes,
+)
+
+
+@ handle_frontend_test(
+    fn_tree="ivy.clone",
+    dtypes_values_casting=st.tuples(
+        st.shared(st.float_arrays(), key="float_clone_arrays"),
+    ),
+    on_device=st.sampled_from(["cpu"]),
+    number_positional_args=1,
+)
+def test_ivy_clone(
+    dtypes_values_casting,
+    on_device,
+    fn_tree,
+    frontend,
+    test_flags,
+):
+
+    x = dtypes_values_casting[0]
```

Comment:
We will need to use the helper function instead of this approach
```python
helpers.test_frontend_function(*args, **kwargs)
```

Reply:
Okay just made some changes working on debuging